feat(changelog): managed-inference-added-new-models-support-nemotron-and 2024-11-28 (#4049)
scaleway/docs-content: changelog/november2024/2024-11-28-managed-inference-added-new-models-support-nemotron-and.mdx

---
title: New models support: Nemotron and Molmo
status: added
author:
    fullname: 'Join the #managed-inference channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-11-28
category: ai-data
product: managed-inference
---

[Llama 3.1 Nemotron 70B](/ai-data/managed-inference/reference-content/llama-3.1-nemotron-70b-instruct/) and [Molmo 72B](/ai-data/managed-inference/reference-content/molmo-72b-0924) are available for deployment on Managed Inference.

> Nemotron improves human-like responses in complex task while Molmo provides increased accuracy on multimodal inputs (text and images).

Reviewer: ```suggestion
Nemotron improves human-like responses in complex tasks, while Molmo provides increased accuracy on multimodal inputs (text and images).
```

---
title: New models support: Nemotron and Molmo
status: added

> author:
>     fullname: ''
>     url: 'https://slack.scaleway.com'

Reviewer 2: ```suggestion
author:
    fullname: 'Join the #managed-inference channel on Slack.'
    url: 'https://slack.scaleway.com'
```

date: 2024-11-28
category: ai-data
product: managed-inference
---

> [Llama 3.1 Nemotron 70B](https://www.scaleway.com/en/docs/ai-data/managed-inference/reference-content/llama-3.1-nemotron-70b-instruct/) and [Molmo 72B](https://www.scaleway.com/en/docs/ai-data/managed-inference/reference-content/molmo-72b-0924) are available for deployment on Managed Inference.
> 
> Nemotron improves human-like responses in complex task while Molmo provides increased accuracy on multimodal inputs (text and images).
> 

Reviewer 2: ```suggestion
[Llama 3.1 Nemotron 70B](/ai-data/managed-inference/reference-content/llama-3.1-nemotron-70b-instruct/) and [Molmo 72B](/ai-data/managed-inference/reference-content/molmo-72b-0924) are available for deployment on Managed Inference.

Nemotron improves human-like responses in complex task while Molmo provides increased accuracy on multimodal inputs (text and images).
```
